src/components/content/deployedServices/common/DeployedServicesHostingType.tsx

```diff
@@ -12,7 +12,7 @@ export function DeployedServicesHostingType({
     currentServiceHostingType,
     className,
 }: {
-    currentServiceHostingType: serviceHostingType;
+    currentServiceHostingType: 'self' | 'service-vendor';
```

Comment:
we must not define the types again. we must read the values only from the API generated classes.